### PR TITLE
jenkins

### PR DESCRIPTION
**Pull request by BasileoLuv**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM centos:7
+ENV JENKINS_HOME=/var/jenkins_home/
+VOLUME /var/jenkins_home/
+RUN yum -y install java
+COPY jenkins.war .
+EXPOSE 8083 8080
```
Comment (savermyas) [inline Dockerfile]: Why do you need to mark port `8083` as exposed?
https://docs.docker.com/engine/reference/builder/#expose

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+CMD ["/usr/bin/java", "-jar", "jenkins.war"] 
+
+
```
Comment (savermyas) [inline Dockerfile]: Too many empty lines

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
-# J-N-D
+# Instruction
+
+- download
```
Comment (savermyas) [inline README.md]: download what?

File: README.md [hunk 4]
```diff
@@ -1 +1,6 @@
...
+- docker build -t "buildname" .
+- docker run -p "port" "buildname"
```
Comment (savermyas) [inline README.md]: which port must I specify here?

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,10 @@
+FROM centos:7
+ENV JENKINS_HOME=/var/jenkins_home/
+VOLUME /var/jenkins_home/
+RUN yum -y install java
+COPY jenkins.war .
```
Comment (savermyas) [inline Dockerfile:5]: you have binary data in you git repo. it is not acceptable.
please, find another way to provide the Jenkins installation into you image build context.

**Review by savermyas — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (BasileoLuv) [reply README.md]: like 8083:8080

File: README.md (hunk 3 above)
Comment (BasileoLuv) [reply README.md]: repo

File: Dockerfile (hunk 2 above)
Comment (BasileoLuv) [reply Dockerfile]: fix it

File: Dockerfile (hunk 1 above)
Comment (BasileoLuv) [reply Dockerfile]: My mistake. Its need it to later project

Comment (savermyas) [reply Dockerfile]: Now you have removed all of them. Why?

File: README.md (hunk 3 above)
Comment (savermyas) [reply README.md]: OK.

File: README.md (hunk 4 above)
Comment (savermyas) [reply README.md]: it doesn't work.
```
GDSpb0659:J-N-D dmyasnikov$ docker run -p "port like 8083:8080" myjenkins
docker: invalid publish opts format (should be name=value but got 'port like 8083:8080').
See 'docker run --help'.
```

Comment (BasileoLuv) [reply README.md]: fix it...

Comment (savermyas) [reply README.md]: I am not supposed to fix it. You are.

**Review by savermyas — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (BasileoLuv) [reply README.md]: Fixed it*

File: Dockerfile (hunk 5 above)
Comment (BasileoLuv) [reply Dockerfile:5]: Now I fixed it. So you can check it

**Comment by savermyas** [comment]
Why hasn't it been merged yet?

**Pull request merged**